libsolidity/formal/Predicate.cpp

```diff
@@ -171,10 +185,13 @@ bool Predicate::isInterface() const
 
 string Predicate::formatSummaryCall(vector<smtutil::Expression> const& _args) const
 {
+	solAssert(isSummary(), "");
+
 	if (auto contract = programContract())
 		return contract->name() + ".constructor()";
 
-	solAssert(isSummary(), "");
+	if (auto funCall = programFunctionCall())
+		return funCall->location().text();
```

Comment:
Since the externally called function can do anything and the counterexample does not rely on any code or arguments, we don't need to evaluate the arguments and just show which piece of code was externally called